meta-ostro-xt/recipes-graphics/xorg-driver/xf86-video-intel_%.bbappend

```diff
@@ -0,0 +1,8 @@
+SRC_URI = "file://always_include_xorg_server.h.patch \
```

Comment:
Where are these patches? The build fails because these cannot be found.

Do you need other than```PACKAGECONFIG``` setting?